app/test_main.py

```diff
@@ -1 +1,70 @@
-# write your code here
+import pytest
+
+from typing import Callable
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+class TestCorrectResults:
+    @pytest.mark.parametrize(
+        "correct_url,"
+        "valid_url_value,"
+        "internet_connection_value,"
+        "expected_access_value",
+        [
+            pytest.param(
+                "https://www.google.com",
```

Comment:
Pass URL as a constant here. Create it at the start of your code.

Example: `CONNECTION_URL = "https://www.google.com"`

```suggestion
                CONNECTION_URL,
```